src/Caching/RedisStorage.php

```diff
@@ -18,6 +18,7 @@ final class RedisStorage implements Storage
 {
 
 	private const NS_NETTE = 'Contributte.Storage';
+	private const NAMESPACE_SEPARATOR = "\x00";
```

Comment:
Can we name it NS_SEPARATOR to prevent confusing that's our constant and not from from nette/caching?

Reply:
Yes. We can 😉